src/commands/executeFunction/eventGrid/ExecuteEventGridFunctionContext.ts

```diff
@@ -0,0 +1,17 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { type IActionContext } from "@microsoft/vscode-azext-utils";
+import { type EventGridSource } from "./eventGridSources";
+
+
+export interface ExecuteEventGridFunctionContext extends IActionContext {
```

Comment:
Similar to what I was saying about `OpenEventGridFileStep`, I think naming this `EventGridExecuteFunctionContext` just makes it easier for other people to associate this with all the other `eventGrid` files.